src/pages/postpage/PostPage.js

```diff
@@ -1,13 +1,92 @@
 import React, {useEffect, useState} from 'react';
 import axios from "axios";
-import BackArrow from "../../assets/back.svg";
 import logo from "../../logo.svg";
 import "./PostPage.css"
+import Header from "../../components/header/Header";
+import Button from "../../components/button/Button";
+import Loader from "../../components/loader/Loader";
+import TopNav from "../../components/topnav/TopNav";
+
+function PostPage() {
```

Comment:
Deze file lijkt heel erg op het HomePage component. Misschien zou je de volgende keer kunnen kijken of je iets wat los kan trekken zodat je het kan hergebruiken